docsets/D/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "D",
-    "version": "2.069.2",
+    "version": "2.093.1",
     "archive": "D.tgz",
```

Comment:
You need to actually update the docset archive. Just changing the version does nothing.